melodic/build_docker.sh

```diff
@@ -1 +1 @@
-docker build -f general_melodic -t general_melodic .
\ No newline at end of file
+docker build --no-cache -f general_melodic -t general_melodic .
```

Comment:
Why using no-cache?

Reply:
Since sometimes I saw changes I made were not applied to a new built image due to the cache, I added to avoid the confusion.

Reply:
I think this is okay then 🤔 
I'll accept it